Template/Pages/About Us.html

```diff
@@ -1,13 +1,11 @@
 <!--
  Title: About Us
  -->
-<p>Staff:</p>
+<h4>Staff:</h4>
```

Comment:
The colon is not part of the title. (You wouldn’t want it included in an auto‐generated table of contents, for example.)

I would just remove it, because the formatting already makes the meaning clear. If you absolutely want to keep it, move it outside the `<h4>` element, and then use CSS to make it look the way you want.